.github/templates/linux_ci_workflow.yml.in

```diff
@@ -195,7 +204,7 @@ jobs:
               ;;
           esac
           echo "SHM_SIZE=${shm_size}" >> "${GITHUB_ENV}"
-      - uses: actions/download-artifact@v2
+      - uses: seemethere/download-artifact-s3@5a1313076ccf40533f94918e2225fe02bd3a96a2
```

Comment:
nit, publish as a formal version? so that future possible small fix doesn't need to have new PRs again

Reply:
We could definitely do that but I also think that having a commit sha for a non-default action makes it a lot easier to debug things when something goes wrong since it points to the specific commit sha